### ISSUE TITLE
Create a wiki page for github-data GHA

### ISSUE THREAD

**Issue by tamara-snyder** [hackforla/website#3283]
### Overview

As a developer, I would like to have up-to-date information about our GitHub actions in a wiki page. For this issue, we will create a Wiki page for the github-data GHA.

### Action Items

- [x] Read the information about the github-data GHA [here](https://github.com/hackforla/website/wiki/HfLA-GitHub-Actions#hack-for-las-ghas)
- [x] Understand how the workflow operates
    - [x] Read through [github-data.json](https://github.com/hackforla/website/blob/gh-pages/_data/external/github-data.json)
    - [x] Read through [get-project-data.js](https://github.com/hackforla/website/blob/gh-pages/github-actions/trigger-schedule/github-data/get-project-data.js)
- [x] Add this new page in alphabetical order on the HfLA GitHub Actions Wiki Page.
- [x] Add the following information for the wiki page 
    - [x] Overview including definitions.
    - [x] What does it do.
    - [x] When does it trigger.
    - [x] Discuss an example of a complete workflow cycle.
    - [x] Other helpful information and related links
- [x] Reach out to the leads during the team meeting or on slack for a review since no PR would be required for this issue.
- [x] For reviewers: Once this issue has been reviewed and approved, please update and edit epic https://github.com/hackforla/website/issues/3229 by
    - [x] Checking off the dependency for this issue
    - [x] If all dependencies are checked off, please move issue https://github.com/hackforla/website/issues/3229 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions

- [HfLA GitHub Actions Wiki Page](https://github.com/hackforla/website/wiki/HfLA-GitHub-Actions#hack-for-las-ghas)
- [Move-closed-issues.yaml](https://github.com/hackforla/website/blob/gh-pages/.github/workflows/move-closed-issues.yaml) 
- [Sort-closed-issues.js](https://github.com/hackforla/website/blob/gh-pages/github-actions/move-closed-issues/sort-closed-issues.js)
- [Move closed issues GitHub action folder](https://github.com/hackforla/website/tree/gh-pages/github-actions/move-closed-issues).

### Updated Wiki page
- [Wiki Page](https://github.com/hackforla/website/wiki/GHA:-github-data)

**Comment by kathrynsilvaconway** [comment]
@JessicaLucindaCheng I was about to put a ready for milestone label on this, but I notice you added the draft label recently. Is there anything else that needs to be done with this one right now?

**Comment by JessicaLucindaCheng** [comment]
@kathrynsilvaconway

> @JessicaLucindaCheng I was about to put a ready for milestone label on this, but I notice you added the draft label recently. Is there anything else that needs to be done with this one right now?

Thanks for checking if the issue is ready for milestone. 😀 I think I put the Draft label because I saw the title has "[DRAFT]" in it but I didn't really read it. However, if you read it and think it is ready for milestone, you can 
- remove [DRAFT] from the title
- remove the Draft label
- then add a ready for milestone label.
- Also, remove the following at the bottom of the issue since there are no dependencies:
> Is there a dependency?
> No
> 
> If Yes, please explain
> No response

**Comment by JessicaLucindaCheng** [comment]
@bishrfaisal I saw you added a milestone and removed the `Ready for Milestone` label. Can this issue be prioritized in the Prioritized Backlog column yet?

**Comment by bishrfaisal** [comment]
@JessicaLucindaCheng Yes, I went ahead and prioritized it in the Prioritized Backlog. Thanks for pointing that out.

**Comment by chrismenke45** [comment]
Availability: Friday 1/27 11am-3pm, Monday-Wednesday Flexible.
ETA: Wednesday 2/1 12pm.

**Comment by chrismenke45** [comment]
Blocker: github-data GHA appears to not be in our repository anymore. This might be an outdated issue as it hasn't been updated for several months? I will ask leadership about this at a meeting.

**Comment by chrismenke45** [comment]
I plan on reviewing my edits in office hours 2/9 7pm.
New ETA: Friday 2/10 3pm
Availability: Wed(2/8)-Thurs(2/9) 11am-3pm

**Comment by MattPereira** [comment]
@chrismenke45 great job communicating with us on this issue. :+1: 

 Apologies about the confusion with the original state of the issue when you first picked it up.

The new [GHA: github-data wiki](https://github.com/hackforla/website/wiki/GHA:-github-data) page looks great! I like how you kept it concise with bullet points. Also appreciate the inclusion of the code block that shows the structure of each repo data object.

I did notice you forgot to include the worflow file that actually calls the `get-project-data.js` file on a schedule so I went ahead and added a link in the wiki for you

- [schedule-daily-1100.yml](https://github.com/hackforla/website/blob/gh-pages/.github/workflows/schedule-daily-1100.yml)
